src/components/sidebar/Sidebar.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+// css
+import './Sidebar.css'
+// static file
+import { sidebarMenu } from 'static/sidebar/sidebarMenu'
+import { NavLink } from 'react-router-dom'
+
+function Sidebar({ children }) {
+  return (
+    <div className='sidebar-HOC'>
+      <aside className='sidebar'>
+        {sidebarMenu.map(menu => <MenuItem
+          key={menu.name}
+          name={menu.name}
+          icon={menu.icon}
+          link={menu.link}
+        />)}
```

Comment:
why not to destructure from menu and then pass them or simple pass menu and destructure it inside the MenuItem component

Reply:
will update in the next iteration 👍